.travis.yml

```diff
@@ -1,15 +0,0 @@
-language:
-    python
-
-python:
-    - 3.6
-    - 3.6-dev
-
-install:
-    - pip3 install -r requirements.txt
-
-script:
-    pytest
-
-notifications:
-    email: false
```

Comment:
We don't have any tests currently, but as mentioned I plan to re-add tests soon on a new branch instead of further cluttering up this branch. The new tests will also include pylint.